esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -428,7 +464,9 @@ define(
     SuperFile: "Super File",
     SuperFile: "Szuperfájl",
     Superfile: "Szuperfálj",
+    Superfiles: "Szuper-fájl",
     SuperFiles: "Szuperfájlok",
```

Comment:
This is duplicate and if we keep it then it would be better to :"Szuper-fájlok" as well.